trunk/SUAVE/Analyses/Aerodynamics/AVL.py

```diff
@@ -1,152 +1,88 @@
+        settings.vsp_mesh_growth_limiting_flag      = False
+                
+        # Build the evaluation process
+        compute = self.process.compute
+        compute.lift = Process()
+
+        # Run SU2 to determine lift
+        compute.lift.inviscid                         = AVL_Inviscid()
+        compute.lift.total                            = SUAVE.Methods.Aerodynamics.AERODAS.AERODAS_setup.lift_total
         
-        # pack conditions
-        state.conditions.aerodynamics.lift_coefficient         = results.conditions.aerodynamics.lift_coefficient
-        state.conditions.aerodynamics.drag_coefficient         = results.conditions.aerodynamics.drag_coefficient
-        state.conditions.aerodynamics.pitch_moment_coefficient = results.conditions.aerodynamics.pitch_moment_coefficient
-
-        return results
-
-
-    def evaluate_conditions(self,run_conditions):
-        """ process vehicle to setup geometry, condititon and configuration
-
-            Inputs:
-                run_conditions - DataDict() of aerodynamic conditions; until input
-                method is finalized, will just assume mass_properties are always as 
-                defined in self.features
-
-            Outputs:
-                results - a DataDict() of type 
-                SUAVE.Analyses.Mission.Segments.Conditions.Aerodynamics(), augmented with
-                case data on moment coefficients and control derivatives
-
-            Assumptions:
-
-        """
+        # Do a traditional drag buildup
+        compute.drag = Process()
+        compute.drag.parasite                      = Process()
+        compute.drag.parasite.wings                = Process_Geometry('wings')
+        compute.drag.parasite.wings.wing           = Methods.Drag.parasite_drag_wing 
+        compute.drag.parasite.fuselages            = Process_Geometry('fuselages')
+        compute.drag.parasite.fuselages.fuselage   = Methods.Drag.parasite_drag_fuselage
+        compute.drag.parasite.propulsors           = Process_Geometry('propulsors')
+        compute.drag.parasite.propulsors.propulsor = Methods.Drag.parasite_drag_propulsor
+        compute.drag.parasite.pylons               = Methods.Drag.parasite_drag_pylon
+        compute.drag.parasite.total                = Methods.Drag.parasite_total
+        compute.drag.induced                       = Methods.Drag.induced_drag_aircraft
+        compute.drag.compressibility               = Process()
+        compute.drag.compressibility.wings         = Process_Geometry('wings')
+        compute.drag.compressibility.wings.wing    = Methods.Drag.compressibility_drag_wing
+        compute.drag.compressibility.total         = Methods.Drag.compressibility_drag_wing_total        
+        compute.drag.miscellaneous                 = Methods.Drag.miscellaneous_drag_aircraft_ESDU
+        compute.drag.untrimmed                     = SUAVE.Methods.Aerodynamics.SU2_Euler.untrimmed
```

Comment:
I think that none of this should be included however I do not know what removing all of these variables will do. We should discuss this in greater detail